.github/workflows/dependabot-pr.yml

```diff
@@ -32,8 +32,8 @@ jobs:
           new_version="${BASH_REMATCH[3]}"
           echo "crate_name: $crate_name, old_version: $old_version, new_version: $new_version"
 
-          ./scripts/cargo-for-all-lock-files.sh -- update -p $crate_name:$old_version --precise $new_version
-          ./scripts/cargo-for-all-lock-files.sh -- tree
+          ./scripts/cargo-for-all-lock-files.sh -- update -p $crate_name:$old_version --precise $new_version || \
+          ./scripts/cargo-for-all-lock-files.sh -- update -p $crate_name --precise $new_version
```

Comment:
thanks for removing `tree`. but, the relaxed variant of `update` is needed? is there some actual failed github action?

Reply:
yeah. like the https://github.com/anza-xyz/agave/pull/1456. dependabot has updated `./Cargo.toml` and `./Cargo.lock`. if we do `./scripts/cargo-for-all-lock-files.sh -- update bytes:1.5.0 --precise 1.6.0`, we will get
```
++ dirname Cargo.lock
+ cd .
+ cargo update bytes:1.5.0 --precise 1.6.0
error: package ID specification `bytes@1.5.0` did not match any packages
Did you mean one of these?

  bytes@1.6.0
```

if we want to update `programs/sbf/Cargo.lock` as well, we need the relaxed command (or we can use tree command instead 🤔 )